esp-hal/src/timer/timg.rs

```diff
@@ -570,6 +513,15 @@ where
     }
 }
 
+impl<T> InterruptConfigurable for Timer<T, Blocking>
```

Comment:
I think we also need an impl for the WDT?

Reply:
Oh true! I added it now but we currently have no way to configure WDT to do anything else but a system reset. I will create a separate issue for that